packages/framework/build.xml

```diff
@@ -693,6 +693,13 @@
 
     <target name="product-search-recreate-structure" depends="product-search-delete-structure,product-search-create-structure" description="Recreates structure for searching via elasticsearch (deletes existing structure and creates new one)"/>
 
+    <target name="product-search-reindex-structure-if-necessary" description="Creates new structure, reindexes it from old one, deletes old structure and add alias to new structure.">
```

Comment:
naming is inconsistent, the phing target says "if necessary", the command says "reindex!" (for me means alwasy), please make them consistent (I prefer the "if necessary" variation)